xclim/indices.py

```diff
@@ -225,16 +226,25 @@ def daily_freezethaw_cycles(tasmax, tasmin, freq='YS'):
 
     The number of days where Tmax > 0℃ and Tmin < 0℃.
     """
+
     ft = (tasmin < K2C) * (tasmax > K2C) * 1
     return ft.resample(time=freq).sum(dim='time')
 
 
 def daily_temperature_range(tasmax, tasmin, freq='YS'):
     r"""Mean of daily temperature range."""
+
     dtr = tasmax - tasmin
     return dtr.resample(time=freq).mean(dim='time')
 
 
+def variable_daily_temperature_range(tasmax, tasmin, freq="YS"):
+    r"""Mean absolute day-to-day variation in daily temperature range"""
```

Comment:
Complete docstring with description, example usage, etc. I've never seen this one used, so worthwhile to explain how it's used.